dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -258,3 +258,105 @@ CWWKD1023.extra.param.explanation=Query conditions that define \
  parameters to supply the values. Extra parameters are not permitted.
 CWWKD1023.extra.param.useraction=Update the repository method signature \
  to remove extra parameters that do not correspond to declared conditions.
+
+CWWKD1024.missing.entity.prop=CWWKD1024E: The {0} method of the {1} repository \
+ interface is missing an entity property name or was supplied with a parameter \
+ that lacks an entity property name. Valid property names for the {2} entity \
+ class are: {3}.
+CWWKD1024.missing.entity.prop.explanation=The query conditions and sort criteria \
+ for a repository method must include an entity property name.
+CWWKD1024.missing.entity.prop.useraction=Inspect the declaration and usage of \
+ the repository method to ensure that an entity property name is supplied for \
+ each query condition and sort criterion.
+
+CWWKD1025.missing.id.prop=CWWKD1025E: The {0} entity class that is used by the \
+ {1} method of the {2} repository interface does not have an Id property or \
```

Comment:
FYI - `Id` is the Java class name of an entity annotation, but I'll get it updated as you recommend because we don't need to refer to the Java class name here and can instead refer to the concept of the ID.